contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1837,6 +1858,11 @@ pltsql_sequence_datatype_map(ParseState *pstate,
 		ereport(WARNING,
 				(errmsg("NUMERIC or DECIMAL type is cast to BIGINT")));
 	}
+
+	if( getBaseType(*newtypid)!=INT2OID ||
```

Comment:
save getBaseType(*newtypid) in a variable, rather than looking up 4 times

Reply:
Done